docs/source/admin/traffic_ops.rst

```diff
@@ -586,6 +590,34 @@ This file sets authentication options for connections to Riak when used as the T
 
 .. impl-detail:: The name of this file is derived from the current database used in the implementation of Traffic Vault - `Riak KV <https://riak.com/products/riak-kv/index.html>`_.
 
+.. _backends.conf:
```

Comment:
it's not hurting anything but this label is unused.
```rst
backends.conf_
```
is a link to a section titled `backends.conf` on the current page, not a link to a label. `:ref:` is used to refer to arbitrary labels from any page/section.